src/wallet/coincontrol.h

```diff
@@ -19,12 +21,12 @@ class CCoinControl
     bool fAllowOtherInputs;
     //! Includes watch only addresses which match the ISMINE_WATCH_SOLVABLE criteria
     bool fAllowWatchOnly;
-    //! Override estimated feerate
+    //! Override automatic min/max checks on fee, m_feerate must be set if true
     bool fOverrideFeeRate;
```

Comment:
Well..  almost
`fOverrideFeeRate` is used from `fundrawtransaction` and it seems to me that it might make sense that you don't want your specified fee rate subject to clamping by various min fees or the maxTxFee when you are using it from there but you do generally.  

In any case it would be a change of functionality to change that, so we should do it separately.